test/functional/feature_coloredcoin.py

```diff
@@ -369,6 +369,7 @@ def run_test(self):
         txSuccess7 = CTransaction()
         txSuccess7.vin.append(CTxIn(COutPoint(coinbase_txs[5].malfixsha256, 0), b""))
         txSuccess7.vout.append(CTxOut(1000, script_reissuable1))
+        txSuccess7.vout.append(CTxOut(4999998000, change_script))
```

Comment:
The code before the change seems to pass this test.

Reply:
Right, it is because there was no change output and the all the leftover TPC was taken as fee. I meant this change to check that TPC less than 1 to be accepted.